test/functional/feature_segwit.py

```diff
@@ -240,7 +240,7 @@ def run_test(self):
         tx.vin.append(CTxIn(COutPoint(int(txid2, 16), 0), b""))
         tx.vout.append(CTxOut(int(49.95 * COIN), CScript([OP_TRUE, OP_DROP] * 15 + [OP_TRUE])))  # Huge fee
         tx.calc_sha256()
-        txid3 = self.nodes[0].sendrawtransaction(ToHex(tx))
+        txid3 = self.nodes[0].sendrawtransaction(ToHex(tx), 0)
```

Comment:
Please use name arguments for unnamed integer values. Also, this commit needs to be squashed into the earlier one (the one where the tests start to fail)

Reply:
Most of the other test code using `sendrawtransaction` doesn't use named parameter for `maxfeerate`.
Do you have any reason for using different convention here?

Reply:
Up to you. Just pick what you prefer.

Reply:
Squashed, and didn't change to a named argument.